src/data/upsert.ts

```diff
@@ -0,0 +1,59 @@
+import { VectorOperationsApi } from '../pinecone-generated-ts-fetch';
+import type { ResponseError } from '../pinecone-generated-ts-fetch';
+import {
+  mapHttpStatusError,
+  PineconeConnectionError,
+  extractMessage,
+} from '../errors';
+import { builOptionConfigValidator } from '../validator';
```

Comment:
**thought** **nit** I suppose we're too far in to worry about fixing this typo. Or am I just bad at English?

Reply:
I noticed this yesterday as well. I didn't want to blow up this diff by changing every file in the repo to fix. Will handle it separately sometime soon, I hope 😂